salesagility/SuiteCRM
# Sort quote and invoice PDF templates by name

## Description

PDF Template pop up results in an unordered list. Cases where several templates are defined, selecting them, forces users to read the complete list. An alphabetical sort is more logical easier to work with. 

## Motivation and Context

Improve the UX of users generating Quotes and Invoices by showing the list in alphabetical order. 

## How To Test This

1. Create several quote and invoice templates

2. Generate a quote and Invoice

3. Print to Pdf for both quote and invoice.

4. Verify that the template are in alphabetical order

## Types of changes




### Final checklist
